components/brave_shields/browser/adblock_stub_response.cc

```diff
@@ -98,6 +100,16 @@ void MakeStubResponse(const network::ResourceRequest& request,
     *data = GetContentForMimeType(response->mime_type);
   }
 
+  if (!redirect.empty()) {
```

Comment:
since this block overwrites values filled before, probably we should put everything above into `else {}`?

Reply:
I considered it but decided against it because mim type might not be specified and the block before gets it.

Reply:
streamed requests through the Fetch API is an example here